ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml

```diff
@@ -255,3 +255,66 @@ SENSOR_ZTM_ZHM_CO2M_VOCM_PMM:
   - CO2M
   - VOCM
   - PMM
+
+SENSOR_ZTM_ZHM_ZOC:
+  description: "Zone occupant counter with zone temperature & humidity monitoring."
+  is_canonical: true
+  implements:
+  - /SENSOR
+  - ZTM
+  - ZHM
+  - ZOC
+
+SENSOR_ZTM_ZHM_CO2M_VOCM_PML_SPLM:
+  description: "Indoor Air Quality sensor with temperature, humidity, CO2, VOC, particulates (density), illuminance and sound monitoring."
+  is_canonical: true
+  implements:
+  - /SENSOR
+  - ZTM
+  - ZHM
+  - CO2M
+  - VOCM
+  - SPLM
+  uses:
+  - zone_air_aqi_sensor
+  - illuminance_sensor
+  - zone_air_pm2pt5_density_sensor
```

Comment:
Implement `PMDM` once you create this new abstract type to capture the particulate matter density fields